presto-main/src/main/java/com/facebook/presto/execution/TaskManagerConfig.java

```diff
@@ -39,6 +39,8 @@
         "sink.new-implementation"})
 public class TaskManagerConfig
 {
+    public static final int DEFAULT_MIN_SPLITS_PER_TASK = 3;
```

Comment:
`SimulationController` shouldn't use this constant.

Instead, a field whose name is along the lines of "guaranteed splits per task" should be added as a simulation parameter to `TaskSpecification`. `TaskExecutionSimulator` should specify it.

In addition, my comment on TaskExecutor would remove usage of this constant there. However, it would then require callers of TaskExecutor constructor come up with a number. I argue that since constructors of TaskExecutor are already using various (and different) literal numbers for runnerThreads and minDrivers, it likely doesn't make sense for them to use this constant.

As a result, the only remaining usage of this constant would be setting the default value of `minDriversPerTask` in this class. Therefore, this constant should probably be inlined.